modules/cluster-logging-collector-log-forward-about.adoc

```diff
@@ -17,7 +17,7 @@ An _output_ is the destination for log data and a pipeline defines simple routin
 An output can be either:
```

Comment:
LogForwarding API had been removed. I think we can drop this file and replace with ClusterLogForwarder